src/main/java/ch/njol/skript/conditions/CondIsPluginEnabled.java

```diff
@@ -0,0 +1,87 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.conditions;
+
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.bukkit.plugin.Plugin;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+@Name("Is Plugin Enabled")
+@Description({"Check if a plugin is enabled/disabled on the server.",
+		"Plugin names can be found in the plugin's 'plugin.yml' file or by using the '/plugins' command, they are NOT the name of the plugin's jar file.",
+		"When checking if a plugin is not enabled, this will return true if the plugin is either disabled or not on the server. ",
+		"When checking if a plugin is disabled, this will return true if the plugin is on the server and is disabled."})
+@Examples({"if plugin \"Vault\" is enabled:", "if plugin \"WorldGuard\" is not enabled:",
+		"if plugins \"Essentials\" and \"Vault\" are enabled:", "if plugin \"MyBrokenPlugin\" is disabled:"})
```

Comment:
Nitpick, but you should make it such that there's only 1 example per line

Reply:
updated 👍